### DISCUSSION TITLE
Will Import-Excel Retrieve Data From a "Protected" Column/Field?

### DISCUSSION THREAD

**Discussion by Chuckles55** [dfinke/ImportExcel#1268] [Q&A]
Hi,
Hidden columns don't seem to be a problem. Just wondering if a "protected" (not encrypted) column will be read.

By the way, this module (Import-Excel) has saved the taxpayers of a certain Pacific Northwest state a LOT of time and money.

**Comment by dfinke** [comment]
Awesome to hear!

Hmm, protected. Not a scenario I tested for. Please post a scenario. Would love to have a look.

Also, would love if you posted one (or more) ways you use the module here https://github.com/dfinke/ImportExcel/discussions/1238

**Comment by Chuckles55** [comment]
Evening,

I did a post on our/the usage of the module. Thanks again for building it. There's real people, who are in serious circumstances, getting indirect benefit from what you took the time to create.

I'll think about how to put something together that captures the essence of what I originally asked about.

Chuck
________________________________
